app/Community/Concerns/ActsAsCommunityMember.php

```diff
@@ -79,14 +81,21 @@ public function followerUsers(): BelongsToMany
         return $this->inverseRelatedUsers()->where('Friendship', '=', UserRelationship::Following);
     }
 
-    public function isFollowing(string $username): bool
+    public function getRelationship(User $user): int
```

Comment:
This has been migrated from the static `UserRelationship::getRelationship()`. It seems more appropriate to have here on the `ActsAsCommunityMember` trait.